tests/src/page_objects/keywords/common.robot

```diff
@@ -98,5 +99,39 @@ Unzip File
 
 Validate and Unzip Test File
     [Arguments]    ${downloaded_file_path}
+    Log To Console    Validating and unziping file
     Validate ZIP File    ${downloaded_file_path}
     Unzip File    ${downloaded_file_path}
+
+Remove Spaces From File Name
+    [Arguments]    ${downloaded_file_path}
+    ${downloaded_file_path_new_name}    Replace String    ${downloaded_file_path}    ${space}    ${empty}
+    Copy File    ${downloaded_file_path}    ${downloaded_file_path_new_name}
+    Wait Until Created    ${downloaded_file_path_new_name}
+    RETURN    ${downloaded_file_path_new_name}
+
+Remove Spaces From Directory Name
+    [Arguments]    ${downloaded_file_path}
+    ${downloaded_file_path_new_name}    Replace String    ${downloaded_file_path}    ${space}    ${empty}
+    Copy Directory   ${downloaded_file_path}    ${downloaded_file_path_new_name}
+    Wait Until Created    ${downloaded_file_path_new_name}
+    Log To Console    Removed spaces from folder name: ${downloaded_file_path_new_name}
+    RETURN    ${downloaded_file_path_new_name}
+
+Compare Directory And Files Tree
+    [Arguments]    ${path_to_unzipped_folder}    ${path_to_unzipped_folder_pattern}
+    ${rc}    ${output}    Run And Return Rc And Output   diff -qr ${path_to_unzipped_folder} ${path_to_unzipped_folder_pattern}
+    Log To Console    rc: ${rc}
```

Comment:
I wonder if these logs are needed - they do not show in the log.html and it is checked two lines after anyway